sympy/integrals/prde.py

```diff
@@ -214,24 +215,24 @@ def poly_linear_constraints(p, d):
 
     return q, M
 
+
 def constant_system(A, u, DE):
     """
     Generate a system for the constant solutions.
 
-    Given a differential field (K, D) with constant field C = Const(K), a Matrix
-    A, and a vector (Matrix) u with coefficients in K, returns the tuple
-    (B, v, s), where B is a Matrix with coefficients in C and v is a vector
-    (Matrix) such that either v has coefficients in C, in which case s is True
-    and the solutions in C of Ax == u are exactly all the solutions of Bx == v,
-    or v has a non-constant coefficient, in which case s is False Ax == u has no
-    constant solution.
+    Given a differential field (K, D) with constant field C = Const(K),
+    a Matrix A, and a vector (Matrix) u with coefficients in K, returns
+    the tuple (B, v), where B is a Matrix with coefficients in C and v
+    is a vector (Matrix) such that either v has coefficients in C, and
+    the solutions in C of Ax == u are exactly all the solutions of
+    Bx == v, or v has a non-constant coefficient, in which case Ax == u
+    has no constant solution.
 
     This algorithm is used both in solving parametric problems and in
-    determining if an element a of K is a derivative of an element of K or the
-    logarithmic derivative of a K-radical using the structure theorem approach.
+    determining if an element of K is a derivative of an element of K
+    or a logarithmic derivative of a K-radical using the structure
+    theorem approach.
 
-    Because Poly does not play well with Matrix yet, this algorithm assumes that
```

Comment:
We have `PolyMatrix` now, so I think we can (I haven't done it) modify the code of `constant_system` to receive input as tuple of Poly's `(poly_num, poly_den)`.